src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/LabeledVariantAnnotationsWalker.java

```diff
@@ -46,7 +48,7 @@
  * walker, performing the operations:
  *
  *   - nthPassApply(n = 0)
- *      - if variant/alleles pass filters and variant-type/overlapping-resource checks, then:
+ *      - if variant/alleles pass filters and variant-type/resource-match checks, then:
```

Comment:
Note that I just changed "overlapping" to "resource-matching" everywhere to distinguish what we're doing here from straightforward genomic overlapping.